charts/posthog/templates/ingress.yaml

```diff
@@ -52,6 +52,18 @@ spec:
   {{- end }}
         paths:
           {{- if (ne (include "ingress.type" .) "clb") }}
+          - pathType: Prefix
```

Comment:
What about the `clb` implementation? Specifically for NGINX we could probably leverage something like the `nginx.ingress.kubernetes.io/server-snippet` annotation (see [docs](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#server-snippet)) and do something like:

```
    nginx.ingress.kubernetes.io/server-snippet: |
      location ~* "^/_metrics" {
          deny all;
          return 403;
        }
```